### PR TITLE
make: add gnubin and gnuman files

### PR DESCRIPTION
**Pull request by robsonpeixoto**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/make.rb [hunk 1]
```diff
@@ -28,6 +28,27 @@ def install
...
+  end
+
+  def caveats
+    if build.without? "default-names" then <<-EOS.undent
+      The command has been installed with the prefix "g".
+      If you do not want the prefix, install using the "with-default-names" option.
+
+      If you need to use these commands with their normal names, you
+      can add a "gnubin" directory to your PATH from your bashrc like:
+        PATH="#{opt_libexec}/gnubin:$PATH"
```
Comment (JCount) [inline Formula/make.rb]: Inserting a newline before this line makes the terminal output more readable

File: Formula/make.rb [hunk 2]
```diff
@@ -28,6 +28,27 @@ def install
...
+
+      Additionally, you can access their man pages with normal names if you add
+      the "gnuman" directory to your MANPATH from your bashrc as well:
+        MANPATH="#{opt_libexec}/gnuman:$MANPATH"
```
Comment (JCount) [inline Formula/make.rb]: Insert a newline before this line as well.

File: Formula/make.rb [hunk 3]
```diff
@@ -28,6 +28,27 @@ def install
...
     system "make", "install"
+
+    if build.without? "default-names"
+      (libexec/"gnubin").install_symlink bin/"gmake" =>"make"
+      (libexec/"gnuman/man1").install_symlink man1/"gmake.1" => "make.1"
+    end
+  end
+
+  def caveats
+    if build.without? "default-names" then <<-EOS.undent
```
Comment (JCount) [inline Formula/make.rb]: Changing this to:
```
    if build.without? "default-names"
      <<-EOS.undent
```
matches `findutils`. (This is a bit of an attempt to create order out of the chaos that has come to be through maintainer inconsistency throughout the years, 🙂) (Feel free to ignore this one 😄)

**Review by JCount — CHANGES_REQUESTED** [review]
Could you also add `revision 1` to the formula below the sha256 hash? Thanks.

File: Formula/make.rb (hunk 3 above)
Comment (robsonpeixoto) [inline Formula/make.rb]: Done in 2f84eb40da8d6113a9271a40c137448182f5fa50
A shameless copy from `findutils` =D

**Review by JCount — APPROVED** [review]

**Comment by JCount** [comment]
@robsonpeixoto Thank you for your contribution to Homebrew!! 🎉

**Comment by robsonpeixoto** [comment]
Thanks @JCount

**Pull request closed without merging**